napari/components/_viewer/model.py

```diff
@@ -134,6 +135,18 @@ def active_markers(self, active_markers):
         self._active_markers = active_markers
         self.events.active_markers(index=self._active_markers)
 
+    @property
+    def key_bindings(self):
+        """dict: custom key bindings
```

Comment:
I think this docstring could be more helpful. I took a crack at it below.
```suggestion
        """dict: custom key bindings. These key bindings are executed after
        		the selected layer's key binding. The dictionary key is
        		a string containing the key pressed and the value is the
        		function to be bound to the key event. The function should
        		accept the viewer object as an input argument.

        		Qt5 key event string docs:
        			https://doc.qt.io/qt-5/qkeyevent.html#text
```

Reply:
You can also specify the typing of the dict with `dict of string:callable`.

Reply:
fixed